blocks/shared-styles/scss/_large-promo.scss

```diff
@@ -9,9 +9,11 @@
     display: inline-block;
     font-size: calculateRem(16px);
     line-height: calculateRem(24px);
-    padding-bottom: map-get($spacers, 'sm');
+    padding-top: map-get($spacers, 'xs');
```

Comment:
looks like this is affecting the desktop breakpoint? in the acceptance criteria, this should only affect the mobile viewport

Reply:
@JackHowa Consolidated the change to affect mobile only.